packages/dm-core-plugins/src/form/components/StyledInputFields.tsx

```diff
@@ -0,0 +1,80 @@
+import styled from 'styled-components'
+import { TextField, Typography } from '@equinor/eds-core-react'
+import React from 'react'
+
+const StyledInputField = styled(TextField)`
+  & :disabled {
+    background: #f7f7f7;
+    color: black;
+  }
+  
+  input {
+    border-radius: 8px;
+  }
+
+  span {
+    color: #6f6f6f;
+  }
+`
+
+const StyledEDSField = (
+  props: React.ComponentProps<typeof StyledInputField> & {
+    isDirty?: boolean
+  }
+) => {
+  return (
+    <StyledInputField
+      {...props}
+      disabled={props.readOnly}
+      readOnly={false}
+      style={
+        props.isDirty && props.variant !== 'error'
+          ? {
+              // @ts-ignore
+              '--eds-input-background': '#85babf5e',
+            }
+          : {}
+      }
+    ></StyledInputField>
+  )
+}
+
+export const StyledTextField = (
+  props: React.ComponentProps<typeof StyledEDSField>
+) => {
+  return (
+    <StyledEDSField
+      {...props}
+      data-testid={`form-text-widget-${props.label}`}
```

Comment:
data-testid=form-text-widget-{...}. I guess this code should be inside the TextWidget or be renamed to form-text-input-{...}? I guess the question is, do we need to have a dedicated data-testid pr widget or not?